server/build/webpack.js

```diff
@@ -33,17 +33,13 @@ function babelConfig (dir, {isServer, dev}) {
   }
 
   const externalBabelConfig = findBabelConfig(dir)
-  if (externalBabelConfig) {
+  if (externalBabelConfig && externalBabelConfig.babelrc) {
     // Log it out once
     if (!isServer) {
       console.log(`> Using external babel configuration`)
-      console.log(`> Location: "${externalBabelConfig.loc}"`)
+      console.log(`> Location: "${externalBabelConfig.babelrc}"`)
     }
-    // It's possible to turn off babelrc support via babelrc itself.
-    // In that case, we should add our default preset.
-    // That's why we need to do this.
-    const { options } = externalBabelConfig
-    mainBabelOptions.babelrc = options.babelrc !== false
+    mainBabelOptions.babelrc = true
   } else {
     mainBabelOptions.babelrc = false
   }
```

Comment:
I'd recommend using the [`createConfigItem`](https://github.com/babel/babel/tree/master/packages/babel-core#babelcreateconfigitemvalue-string----function--string----function---void--dirname-string-type-preset--plugin--configitem) utility on the `mainBabelOptions.presets.push(require.resolve('./babel/preset'))` line below this, rather than just passing the string, otherwise this will end up doing a lot more work.